mlflow/tensorflow/__init__.py

```diff
@@ -1087,10 +1177,11 @@ def _patch_implementation(
                     # & tf.keras callbacks if necessary
                     callbacks = list(kwargs.get("callbacks") or [])
                     kwargs["callbacks"], self.log_dir = _setup_callbacks(
-                        callbacks, log_models, metrics_logger
+                        callbacks, False, metrics_logger
```

Comment:
along with line 1107, both calls to `_setup_callbacks()` have the 2nd argument as 'False'. Can we remove this dead code since all 4 usages of that function are now set to a bool False constant?

Reply:
cleaned up in the last commit. thanks!